runbotics/runbotics-scheduler/src/scheduler-database/process/process.controller.ts

```diff
@@ -193,6 +194,25 @@ export class ProcessController {
         @Pageable() paging: Paging,
         @UserDecorator() user: User,
     ) {
+        // @ts-expect-error property not in built-in type
+        const { _type, _value } = specs.where.processCollectionId?.valueOf()?.value.at(0) ?? {};
+
+        if (_type === 'isNull' || (_type === 'equal' && _value.trim())) {
+           return this.processCrudService.getPage(user, specs, paging);
```

Comment:
Could this be handled in criteria class?

Reply:
we'll consider it after process-collection migration